### PR TITLE
Explore main page for mobile

### PR DESCRIPTION
**Pull request by fcavuoti**
Implementing explore main page for mobile by changing the filters layout

### CODE REVIEW

File: client/src/components/filters/filterMobile.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const Word = styled.div`
+    font-size:18px;
+`;
+
+const SignWrapper = styled.div`
+    font-size: 22px;
+    margin-right: 10px;
+`;
+
+const blue = '#42B7CB';
```
Comment (erin2722) [inline client/src/components/filters/filterMobile.js:41]: for the colors here, could we use theme provider?

File: client/src/components/filters/filterMobile.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+const grey = "#9A9A9A";
+const text = 'Filters';
+const plus = '+';
+
+export default class FilterMobile extends React.Component {
```
Comment (erin2722) [inline client/src/components/filters/filterMobile.js:46]: can we switch this to hooks instead of class components?

**Review by erin2722 — CHANGES_REQUESTED** [review]
Two small codebase-style changes, but otherwise this looks incredible!!!

Also, when you repush can you pull from frontend first to resolve the conflicts?

This is awesome, and thank you! 👍 🎉

**Review by lauracastrovenegas — CHANGES_REQUESTED** [review]
Looks great!! 👏🏻👏🏻 Great job with the state. Looks very clean 😁 It is ready to go and merge after just one small change.

I know it's a little irrelevant to what you were doing but could you go through the styling for the explore page and the shuffle.js and change the hardcoded color values to use props:` ${(props) => props.theme.colors.white};`? I pointed out a few spots on the explore page but there are a few in the shuffle.js file as well. Just a quick fix that we can include in our merge, killing two birds with one stone.

Otherwise, looks amazing!! Once you push those styling prop changes I will merge!

**Review by lauracastrovenegas — APPROVED** [review]
WOW! So great that you were able to convert those components to hooks! Really great job!! 😁🎉

**Pull request merged**